ci/after_success.sh

```diff
@@ -2,13 +2,17 @@
 # Generate coverage report
 ./gradlew jacocoRootReport coveralls
 
-# Publish Javadoc
+# Tasks to do only on an actual release
```

Comment:
I thought after_success was triggered after each travis Run, would it be better to move this portion out to a separate script if it's going to be run manually?

Reply:
I was thinking to have the release to Maven Central and updating of javadocs be automatically done via Travis. With these conditionals, this would only happen on a merge to master.

Reply:
A merge to master shouldn't trigger a release tho. E.g. it's weird to publish a new jar/javadocs if we change a Readme.

Reply:
Hmm makes sense.